### PR TITLE
Fix circle ci macos

### PR DESCRIPTION
**Pull request by BertrandD**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@ jobs:
                 ~/cmake-3.16.5-Linux-x86_64/bin/ctest --output-on-failure -timeout 120
   build_macos_release:
     macos:
-      xcode: "12.0.0"
+      xcode: "12.5.1"
```
Comment (jcoatelen-ledger) [inline .circleci/config.yml:107]: If i'm not mistaken, there will be a new version to come, see with Victor about that, for reference: https://github.com/LedgerHQ/lib-ledger-core-build-env/pull/9

**Review by jcoatelen-ledger — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (viktorb-ledger) [reply .circleci/config.yml:107]: @jcoatelen-ledger
How that build env update can fix xcode version?

Comment (jcoatelen-ledger) [reply .circleci/config.yml:107]: it's not related, I made a confusion sorry

**Review by jcoatelen-ledger — APPROVED** [review]
LGTM

**Pull request merged**